examples/jsm/renderers/webgpu/nodes/WebGPUNodeBuilder.js

```diff
@@ -661,7 +661,7 @@ class WebGPUNodeBuilder extends NodeBuilder {
 
 	}
 
-	getMethod( method, shaderStage = this.shaderStage ) {
+	getMethod( method ) {
```

Comment:
`getMethod()` only needs a single parameter.

Reply:
Okay. I think that this argument became unnecessary after some internal refactoring.